websites/management/commands/unpublish_sites.py

```diff
@@ -0,0 +1,125 @@
+"""Unpublish course webpages provided to the command that have no dependencies. If there are dependencies, print the dependencies."""
+import json
+
+from django.conf import settings
+from django.db.models import Q
+
+from content_sync.api import trigger_unpublished_removal
+from main.management.commands.filter import WebsiteFilterCommand
+from users.models import User
+from websites.constants import (
+    CONTENT_TYPE_COURSE_LIST,
+    CONTENT_TYPE_PAGE,
+    CONTENT_TYPE_RESOURCE_COLLECTION,
+    PUBLISH_STATUS_NOT_STARTED,
+)
+from websites.models import Website, WebsiteContent
+from websites.serializers import WebsiteBasicSerializer, WebsiteContentSerializer
+
+
+class Command(WebsiteFilterCommand):
+    """Unpublish course webpages provided to the command that have no dependencies. If there are dependencies, print the dependencies."""
+
+    help = __doc__
+
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+        parser.add_argument(
```

Comment:
`WebsiteFilterCommand` already includes an optional `--filter` argument for sites (comma-delimited list of either names or short_ids), you can either use that instead of adding `--course_ids` (which might be best for consistency's sake), or not inherit from `WebsiteFilterCommand` (ie `class Command(BaseCommand):`)

Reply:
Updated in 9b2e6ed52d822fba85c459bcc7e67aeb81be9400.